source/utils/types.ts

```diff
@@ -47,8 +49,20 @@ export interface InterfaceWithDefaults extends Instance {
 	};
 }
 
+interface RetryDescriptor {
+	retries: ((retry: number, error: Error) => number) | number;
+	methods: Method[];
+	statusCodes: number[];
+	errorCodes: string[];
+}
```

Comment:
Make sure it has everything from the docs: https://github.com/sindresorhus/got#retry